pkg/kubelet/cm/container_manager_stub.go

```diff
@@ -96,11 +98,11 @@ func (cm *containerManagerStub) GetDevicePluginResourceCapacity() (v1.ResourceLi
 }
 
 func (m *podContainerManagerStub) GetPodCgroupConfig(_ *v1.Pod, _ v1.ResourceName) (*ResourceConfig, error) {
-	return nil, nil
+	return nil, fmt.Errorf("not implemented")
```

Comment:
I didn't realize these methods were just added for the VPA feature...

I agree it seems better to return errors instead of `nil, nil` which can easily cause panics, but the fact that windows is using this stub impl makes me wonder what error loops windows nodes will get caught in or log profusely over

Reply:
Windows is not supported for alpha, we decided a while back that it is not alpha blocker. There is another PR that seeks to add support for Windows: https://github.com/kubernetes/kubernetes/pull/112599/files# , it needs some more work.
/cc @marosset